src/components/header/header.jsx

```diff
@@ -0,0 +1,26 @@
+import { useState } from "react"
+import Login from "../login/login"
+import UserBlock from "../userBlock/userBlock"
+import "./style.css"
+
+export const Header = () => {
+    const [loginVisible, setLoginVisible] = useState('none')
+
+    return (
+        <div>
+            <div className="header">
+                <div className="buttonBlock">
+                    <button text='Русский' hover-text='Switch to English' className="buttonLng"></button>
+                </div>
+                <h1>Instapro</h1>
+                <div className="buttonBlock">
+                    <button onClick={() => {loginVisible === 'none' ? setLoginVisible('flex') : setLoginVisible('none')}} className="buttonEnter">Enter</button>
```

Comment:
css значения в js использовать не оч хорошая идея, лучше использовать булев флаг.

Так, если ты захочешь поменять логику для "видимость логина", тебе не придется бегать по всем js файлам и исправлять css, а можно будет исправить только целевую логику

```suggestion
                    <button onClick={() => {loginVisible === 'none' ? setLoginVisible(true) : setLoginVisible(false)}} className="buttonEnter">Enter</button>
```

Reply:
![image](https://github.com/VolkOligarx/Insta-Like-Interface/assets/113520935/5aa36fdc-f9c5-4d07-8304-1198c558ea1e)
Я правильно перенес данную логику? 
И уточнение, это сделано, что бы при смене логики менять только параметры в header? Стоит ли мне в след разы максимально переносить логику в дочерние компоненты?

Reply:
Да, чем ближе логика к месту использования тем лучше. 
На скринах вроде правильно, над смотреть ближе